sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/cosmos/properties/AzureCosmosPropertiesConfiguration.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.implementation.cosmos.properties;
+
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Import;
+
+@Import({
+    ConfigurationWithClassWithBean.class,
+    ConfigurationWithClassWithoutBean.class,
+    ConfigurationWithoutClass.class
+})
+@EnableConfigurationProperties
+public class AzureCosmosPropertiesConfiguration {
```

Comment:
```suggestion
class AzureCosmosPropertiesConfiguration {
```

Reply:
The "public" can't be deleted, because it will be used outside "properties" package.